tools/provision/lib.sh

```diff
@@ -188,6 +188,34 @@ function install_yara() {
   fi
 }
 
+function install_openssl() {
+  SOURCE=openssl-1.0.2d
+  TARBALL=$SOURCE.tar.gz
+  URL=http://openssl.org/source/$TARBALL
+
+  if provision openssl /usr/lib/libssl.so.1.0.0; then
```

Comment:
Thanks for updating the `--prefix`, you'll need to modify this check to -> `/usr/local/lib/libssl.so.1.0.0`. Builds will pass without the update but they will always reinstall since this may never exist.